lib/kitchen/driver/ec2.rb

```diff
@@ -98,10 +98,10 @@ class Ec2 < Kitchen::Driver::SSHBase
         end
       end
 
-      def self.iam_creds
+      def iam_creds
         @iam_creds ||= begin
-          fetch_credentials(use_iam_profile:true)
-        rescue RuntimeError => e
+          fetch_credentials(use_iam_profile: true)
+        rescue RuntimeError, NoMethodError => e
```

Comment:
I still have the same question about `NoMethodError` - why is it needed?  After including the ServiceMethods, you shouldn't ever encounter a `NoMethodError`.  Or am I missing something?